GLMakie/src/screen.jl

```diff
@@ -686,7 +687,7 @@ function Makie.colorbuffer(screen::Screen, format::Makie.ImageStorageFormat = Ma
     ctex = screen.framebuffer.buffers[:color]
     # polling may change window size, when its bigger than monitor!
     # we still need to poll though, to get all the newest events!
-    # GLFW.PollEvents()
+    pollevents(screen)
     # keep current buffer size to allows larger-than-window renders
     render_frame(screen, resize_buffers=false) # let it render
```

Comment:
I added `pollevents()` here so lighting updates from `render_tick` can take effect before drawing the frame. Given the note I would assume this breaks output sizes with `save` and `record`, however I have no problems saving/recording at `resolution = (4000, 4000)`.